libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -59,9 +59,11 @@ +(instancetype)sharedInstance {
 - (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil {
     self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
     if (self) {
-        _navBarColor = [UIColor colorWithRed:22.0/255.0 green:87.0/255.0 blue:205/255.0 alpha:1.0];
+        _navBarColor = [UIColor colorWithRed:0 green:0.439 blue:0.824 alpha:1];
```

Comment:
Like this http://stackoverflow.com/questions/2824187/objective-c-defining-uicolor-constants